src/util/mw-api.ts

```diff
@@ -259,7 +259,7 @@ export function mwRetToArticleDetail(obj: QueryMwRet): KVS<ArticleDetail> {
 
 export async function checkApiAvailability(url: string, loginCookie = ''): Promise<boolean> {
   try {
-    const resp = await axios.get(url, { maxRedirects: 0, headers: { cookie: loginCookie } })
+    const resp = await axios.get(decodeURI(url), { maxRedirects: 0, headers: { cookie: loginCookie } })
```

Comment:
Decoding URL should help to avoid problems with querying to MW Action API for different foreign wikis.